cmd/nerdctl/run_windows.go

```diff
@@ -19,9 +19,12 @@ package main
 import (
 	"context"
 	"fmt"
+	"strconv"
 
 	"github.com/containerd/containerd/containers"
 	"github.com/containerd/containerd/oci"
+	"github.com/containerd/containerd/pkg/cri/annotations"
```

Comment:
We can just redefine the constants without adding extra dependency on cri

Reply:
Yes, definitely makes sense to do that along with maybe updating to using hcsshim once a released package has these constants exported since we already rely on it